extras/indexing/src/main/java/org/apache/rya/indexing/mongodb/AbstractMongoIndexer.java

```diff
@@ -73,12 +72,12 @@
     protected T storageStrategy;
 
     private MongoDbBatchWriter<DBObject> mongoDbBatchWriter;
+    protected String collectionName;
```

Comment:
Is this field ever set anywhere?

Reply:
Oh, the subclasses change it directly. That's unclear, why not leave it the way it was?

Reply:
yes, the implementations of AbstractMongoIndexer set it in the init() function.  I can't have it in the constructor because of the hadoop/reflexive construction

Reply:
How did it work before, then?

Reply:
getCollectionName()  would build the collection name with the collection suffix and rya prefix